target-service/src/main/java/org/springframework/cloud/appbroker/samples/targetservice/CustomSpaceService.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.samples.targetservice;
+
+public class CustomSpaceService {
+
+	String retrieveSpaceName() {
+		return "my-space";
```

Comment:
🤔 "myspace"

Reply:
Old memories 😆 
Changed it to `customspace` just in case.

Reply:
Old memories 😆 
Changed it to `customspace` just in case.